addon/services/translation.js

```diff
@@ -0,0 +1,12 @@
+import Ember from "ember";
+
+export default Ember.Service.extend({
+  i18n: Ember.inject.service(),
+  intl: Ember.inject.service(),
+  translationServices: Ember.computed.collect("i18n", "intl"),
+  hasTranslationService: Ember.computed.notEmpty("translationServices"),
+  activeTranslationServices: Ember.computed("translationServices", function() {
+    return this.get("hasTranslationService") ? Ember.A() : this.get("translationServices").compact();
```

Comment:
Shouldn't this be exactly opposite? An empty array when there are is no translation service?